shell/src/main/java/tachyon/shell/command/CopyFromLocalCommand.java

```diff
@@ -205,4 +205,9 @@ private void copyPath(File src, TachyonURI dstPath) throws IOException {
   public String getUsage() {
     return "copyFromLocal <src> <remoteDst>";
   }
+
+  @Override
+  public String getDescription() {
+    return "Copies the file specified by <src> to the path specified by <remoteDst>.";
```

Comment:
`Copies the file from local filesystem to Tachyon filesystem`, to be consistent with `copyToLocal`'s description.